src/ORM/Table.php

```diff
@@ -1531,7 +1533,7 @@ public function get($primaryKey, array $options = []): EntityInterface
                     'get-%s-%s-%s',
                     $this->getConnection()->configName(),
                     $this->getTable(),
-                    json_encode($primaryKey)
+                    str_replace('"', '', json_encode($primaryKey))
```

Comment:
This is quite hackish but I didn't want to ensure the key doesn't change for integer ids.